src/EnergyPlus/DataHeatBalFanSys.hh

```diff
@@ -89,6 +89,15 @@ namespace DataHeatBalFanSys {
         int DualPMVErrIndex = 0; // Dual PMV setpoint error index
     };
 
+    struct SurfQRadFromHVACData
```

Comment:
New struct to hold the old Array1Ds for SurfQHTRadSys, SurfQHWBaseboard, etc.

Reply:
:) :) :)